canonical/charmhub.io
# WD-658 filter interfaces by status

## Done
- Filter interfaces based on status of 'Live'
## How to QA
- Go to /interfaces.json
- Confirm that all returned interface has {status: "Live"}
## Issue / Card
Fixes #

## Screenshots
